frontend/src/scenes/teamActivityDescriber.tsx

```diff
@@ -139,6 +139,48 @@ const teamActionsMapping: Record<
             ],
         }
     },
+    survey_config: (change: ActivityChange | undefined): ChangeMapping | null => {
+        const before = change!.before as TeamSurveyConfigType
+        const after = change!.after as TeamSurveyConfigType
+        const descriptions = []
+        const preamble = 'Survey Configuration : '
+        if (before === undefined) {
+            descriptions.push('Survey Configuration was enabled')
+        }
+
+        const propertyChangeDesc = (
+            name: string,
+            callback: (config: TeamSurveyConfigType) => string | undefined
+        ): void => {
+            if (callback(before) !== callback(after)) {
+                descriptions.push(`${preamble} ${name} was changed from "${callback(before)}" to "${callback(after)}"`)
+            }
+        }
+
+        if (before?.appearance?.whiteLabel !== after?.appearance?.whiteLabel) {
+            descriptions.push(
+                `${preamble} Survey White labeling was ${after?.appearance?.whiteLabel ? 'enabled' : 'disabled'}`
+            )
+        }
+
+        if (before?.appearance?.displayThankYouMessage !== after?.appearance?.displayThankYouMessage) {
+            descriptions.push(
+                `${preamble} displayThankYouMessage was ${after?.appearance?.whiteLabel ? 'enabled' : 'disabled'}`
+            )
+        }
+
+        propertyChangeDesc('backgroundColor', (c) => c?.appearance?.backgroundColor)
+        propertyChangeDesc('submitButtonColor', (c) => c?.appearance?.submitButtonColor)
+        propertyChangeDesc('submitButtonTextColor', (c) => c?.appearance?.submitButtonTextColor)
+        propertyChangeDesc('ratingButtonColor', (c) => c?.appearance?.ratingButtonColor)
+        propertyChangeDesc('ratingButtonActiveColor', (c) => c?.appearance?.ratingButtonActiveColor)
+        propertyChangeDesc('borderColor', (c) => c?.appearance?.borderColor)
+        propertyChangeDesc('placeholder', (c) => c?.appearance?.placeholder)
+        propertyChangeDesc('thankYouMessageHeader', (c) => c?.appearance?.thankYouMessageHeader)
+        propertyChangeDesc('position', (c) => c?.appearance?.position)
```

Comment:
I don't like how these messages say "Team Survey Config"; it's too casual, not clear, and the Capitalized Text is confusing.  
<img width="588" alt="image" src="https://github.com/user-attachments/assets/8a995bd2-7842-4f81-852b-69514a53d92b">

Can you change the `Team Survey Config` reference to something more user-friendly?  I'd like to say something like "set ${item} to ${value} for global survey appearance/styles"